index.bs

```diff
@@ -5840,10 +5840,13 @@ The attestation certificate MUST have the following fields/extensions:
 
 - The Basic Constraints extension MUST have the CA component set to [FALSE].
 
-- An Authority Information Access (AIA) extension with entry `id-ad-ocsp` and a CRL Distribution Point extension [[RFC5280]]
-    are both OPTIONAL as the status of many attestation certificates is available through authenticator metadata services.
-    See, for example, the FIDO Metadata Service [[FIDOMetadataService]].
+Additionally, Authority Information Access (AIA) extension with entry `id-ad-ocsp` and a CRL Distribution Point extension 
+    [[RFC5280]] are both OPTIONAL as the status of many attestation certificates is available through authenticator metadata
+    services. See, for example, the FIDO Metadata Service [[FIDOMetadataService]].
 
+The firmware of a particular authenticator model MAY be differentiated using the Extension OID `1.3.6.1.4.1.45724.1.1.5` 
+    (`id-fido-gen-ce-fw-version`). When present, this attribute contains an unsigned INTEGER which is incremented for new
+    firmware release versions. The extension MUST NOT be marked as critical.
```

Comment:
The encoding of X.509 extensions is a pitfall I've seen many ways to fall into, so I think an example encoded extension structure like the one for `id-fido-gen-ce-aaguid` would be good here.

Reply:
Do you think this is important for an INTEGER attribute? We do not give such an example for Version.

Reply:
As mentioned on today's WG call, one joint example with all of the attributes should do - we don't need individual examples of each on its own.